stable/cockroachdb/templates/cockroachdb-statefulset.yaml

```diff
@@ -173,11 +173,16 @@ spec:
   volumeClaimTemplates:
   - metadata:
       name: datadir
-      annotations:
-        volume.alpha.kubernetes.io/storage-class: "{{ .Values.StorageClass }}"
     spec:
       accessModes:
         - "ReadWriteOnce"
       resources:
         requests:
           storage: "{{ .Values.Storage }}"
+{{- if .Values.StorageClass }}
+{{- if (eq "-" .Values.StorageClass) }}
+  storageClassName: ""
```

Comment:
This is at the wrong indent level, it should be under `spec`

Reply:
My bad, thanks @prydonius